nilearn/plotting/img_plotting.py

```diff
@@ -305,6 +307,9 @@ def plot_img(img, cut_coords=None, output_file=None, display_mode='ortho',
         Default=None.
     %(vmin)s
     %(vmax)s
+    radiological : boolean, optional
```

Comment:
It looks like this docstring is copy-pasted a few times between functions. Could you add it to `nilearn/_utils/docs.py`, and then replace this by `%(radiological)s`?

Reply:
Still relevant. Let me know if this isn't clear.